code/modules/antagonists/cult/runes.dm

```diff
@@ -284,11 +284,16 @@ structure_check() searches for nearby cultist structures required for the invoca
 	if(!C)
 		return
 
+	if(ispAI(sacrificial))
```

Comment:
Shouldn't this be in `is_convertable_to_cult()` instead?

Reply:
convertible != sacrificable

Reply:
Did re-read the code and you're actually correct, my bad.
Ideally this would be a living flag instead of an istype check, but won't demand such a refactor when this fixes it.